src/tox/config/__init__.py

```diff
@@ -1801,35 +1801,32 @@ def _replace_match(self, match):
         if not any(g.values()):
             return os.pathsep
 
+        sub_type = g["sub_type"]
+        default_value = g["default_value"]
+
         if sub_value == "posargs":
-            return self.reader.getposargs(match.group("default_value"))
+            return self.reader.getposargs(default_value)
 
-        try:
```

Comment:
https://github.com/tox-dev/tox/pull/1715 is the prefered approach for this chunk.

I'll rebase after that is merged, assuming it is accepted.

Reply:
Rebased; im ambivalent about whether this is merged or not.  Also it feels odd writing a changelog entry for this.  I'd be fine with slipping this into another PR to avoid it being a standalone changelog entry.